htdocs/API/2.0/index.php

```diff
@@ -36,5 +37,6 @@
 $server->addClass(ApiForumController::class, '/forum');
 $server->addClass(ApiLedenController::class, '/leden');
 $server->addClass(ApiMaaltijdenController::class, '/maaltijden');
+$server->addClass(ApiSponsorkliksController::class, '/sponsorkliks');
```

Comment:
Ik weet niet helemaal zeker of deze plek slim is. De sponsorkliks api heeft helemaal niets te maken met de mobiele api. Misschien is het beter om het in `/sponsorkliks/api/v1/affiliates` (o.i.d.) te stoppen. De enige reden waarom deze code in deze repo staat is omdat het dan makkelijk online te zetten is, het lijkt me dan wel beter om het op z'n eigen plek te zetten.

`API/2.0` is ook nog redelijk aan veranderingen onderhevig en wordt misschien wel ~weggegooid~gedeprecate als de stek goed op mobiel werkt.

Je zou ook gewoon `sponsorkliks.json` naar `../htdocs/sponsorkliks/api/v1/affiliates.json` kunnen schrijven dan.

Reply:
Momenteel is het inderdaad vooral een mobiele API, maar toch lijkt het me beter om, als we dan toch dingen via API exposen om dat consistent op dezelfde manier te doen. Mochten we de API willen deprecaten kunnen we er altijd nog voor kiezen om deze json op dezelfde endpoint aan te bieden.
Ik heb bewust niet gekozen om het bestand naar htdocs te schrijven, aangezien dat onder versiebeheer staat en data niet.